assets/xml/objects/object_tab.xml

```diff
@@ -1,63 +1,63 @@
+        <Texture Name="gBartenFootTex" OutName="barten_foot" Format="rgba16" Width="16" Height="16" Offset="0x7A28" />
         <!-- <Blob Name="object_tab_Blob_007C28" Size="0x220" Offset="0x7C28" /> -->
-        <Limb Name="object_tab_Standardlimb_007E48" Type="Standard" EnumName="OBJECT_TAB_LIMB_01" Offset="0x7E48" />
-        <Limb Name="object_tab_Standardlimb_007E54" Type="Standard" EnumName="OBJECT_TAB_LIMB_02" Offset="0x7E54" />
-        <Limb Name="object_tab_Standardlimb_007E60" Type="Standard" EnumName="OBJECT_TAB_LIMB_03" Offset="0x7E60" />
-        <Limb Name="object_tab_Standardlimb_007E6C" Type="Standard" EnumName="OBJECT_TAB_LIMB_04" Offset="0x7E6C" />
-        <Limb Name="object_tab_Standardlimb_007E78" Type="Standard" EnumName="OBJECT_TAB_LIMB_05" Offset="0x7E78" />
-        <Limb Name="object_tab_Standardlimb_007E84" Type="Standard" EnumName="OBJECT_TAB_LIMB_06" Offset="0x7E84" />
-        <Limb Name="object_tab_Standardlimb_007E90" Type="Standard" EnumName="OBJECT_TAB_LIMB_07" Offset="0x7E90" />
-        <Limb Name="object_tab_Standardlimb_007E9C" Type="Standard" EnumName="OBJECT_TAB_LIMB_08" Offset="0x7E9C" />
-        <Limb Name="object_tab_Standardlimb_007EA8" Type="Standard" EnumName="OBJECT_TAB_LIMB_09" Offset="0x7EA8" />
-        <Limb Name="object_tab_Standardlimb_007EB4" Type="Standard" EnumName="OBJECT_TAB_LIMB_0A" Offset="0x7EB4" />
-        <Limb Name="object_tab_Standardlimb_007EC0" Type="Standard" EnumName="OBJECT_TAB_LIMB_0B" Offset="0x7EC0" />
-        <Limb Name="object_tab_Standardlimb_007ECC" Type="Standard" EnumName="OBJECT_TAB_LIMB_0C" Offset="0x7ECC" />
-        <Limb Name="object_tab_Standardlimb_007ED8" Type="Standard" EnumName="OBJECT_TAB_LIMB_0D" Offset="0x7ED8" />
-        <Limb Name="object_tab_Standardlimb_007EE4" Type="Standard" EnumName="OBJECT_TAB_LIMB_0E" Offset="0x7EE4" />
-        <Limb Name="object_tab_Standardlimb_007EF0" Type="Standard" EnumName="OBJECT_TAB_LIMB_0F" Offset="0x7EF0" />
-        <Limb Name="object_tab_Standardlimb_007EFC" Type="Standard" EnumName="OBJECT_TAB_LIMB_10" Offset="0x7EFC" />
-        <Limb Name="object_tab_Standardlimb_007F08" Type="Standard" EnumName="OBJECT_TAB_LIMB_11" Offset="0x7F08" />
-        <Limb Name="object_tab_Standardlimb_007F14" Type="Standard" EnumName="OBJECT_TAB_LIMB_12" Offset="0x7F14" />
-        <Limb Name="object_tab_Standardlimb_007F20" Type="Standard" EnumName="OBJECT_TAB_LIMB_13" Offset="0x7F20" />
-        <Skeleton Name="object_tab_Skel_007F78" Type="Flex" LimbType="Standard" LimbNone="OBJECT_TAB_LIMB_NONE" LimbMax="OBJECT_TAB_LIMB_MAX" EnumName="ObjectTabLimb" Offset="0x7F78" />
-        <Animation Name="object_tab_Anim_0086AC" Offset="0x86AC" />
+        <Limb Name="gBartenPelvisLimb" Type="Standard" EnumName="BARTEN_PELVIS_LIMB" Offset="0x7E48" />
+        <Limb Name="gBartenRightThighLimb" Type="Standard" EnumName="BARTEN_RIGHT_THIGH_LIMB" Offset="0x7E54" />
```

Comment:
```suggestion
        <Limb Name="gBartenPelvisLimb" Type="Standard" EnumName="BARTEN_LIMB_PELVIS" Offset="0x7E48" />
        <Limb Name="gBartenRightThighLimb" Type="Standard" EnumName="BARTEN_LIMB_RIGHT_THIGH" Offset="0x7E54" />
```
The styling for the `LIMB` enum is to have `ACTOR_LIMB_XXX`, could you move `LIMB` to be earlier in these names?